postman/MIDAZ.postman_collection.json

```diff
@@ -2404,7 +2692,14 @@
 					"name": "Accounts -> Operation by id",
 					"request": {
 						"method": "GET",
-						"header": [],
+						"header": [
+							{
+								"key": "Midaz-Id",
```

Comment:
what is the correct name? **midaz-ID**, **midaz-Id** or **midaz-id**???? please use only one.